at-chuongphu/excercise5.rb

```diff
@@ -0,0 +1,41 @@
+# ex1
+class Student
+	attr_accessor :first_name, :last_name, :list_course
+  def initialize
+    @list_course = []
+  end
+  def first_name
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/1c0a5adcb251976aac9f1128795ec3dbcba91c88/at-chuongphu/excercise5.rb#L7"></span>Lint/DuplicateMethods: Method <code>Student#first_name</code> is defined at both at-chuongphu/excercise5.rb:3 and at-chuongphu/excercise5.rb:7.</td>
    </tr>
  </tbody>
</table>